packages/cms-lib/logger.js

```diff
@@ -37,6 +39,12 @@ class Logger {
   log(...args) {
     console.log(...args);
   }
+  success(...args) {
+    console.log(...stylize('[SUCCESS]', Styles.success, args));
+  }
+  info(...args) {
+    console.log(...stylize('[INFO]', Styles.info, args));
```

Comment:
```suggestion
    console.info(...stylize('[INFO]', Styles.info, args));
```

https://developer.mozilla.org/en-US/docs/Web/API/Console/info